.travis.yml

```diff
@@ -9,8 +9,8 @@ jdk:
   - oraclejdk8
   - oraclejdk9
 env:
-  - SERVER_VERSION=wildfly1010
-  - SERVER_VERSION=wildfly1000
+ ## - SERVER_VERSION=wildfly1010
+ ## - SERVER_VERSION=wildfly1000
```

Comment:
why are you completely commenting out the 1010 and 1000 runs? This way only 1100 runs are executed